src/components/ReportPage/index.jsx

```diff
@@ -119,14 +119,21 @@ class ReportPage extends PureComponent {
     }
   }
 
+  capitalizeFirst = (value) => {
+    const newValue = value ? value[0].toUpperCase() + value.slice(1) : undefined;
+    return newValue;
+  }
+
   doSearch = (searchValue, optionId) => {
+    const newSearchValue = searchValue ? searchValue.toLowerCase() : undefined;
+    const finalSearchValue = this.capitalizeFirst(newSearchValue);
     const { reportData } = this.state;
     let filteredReport = null;
-    if (searchValue && optionId === 1) {
-      filteredReport = reportData.filter(report => report.fellowName === searchValue);
+    if (finalSearchValue && optionId === 1) {
+      filteredReport = reportData.filter(report => report.fellowName === finalSearchValue);
```

Comment:
I believe 
```
filteredReport = reportData.filter(
	report => report.fellowName.toLowerCase() === searchValue.toLowerCase()
);
```
can help us avoid creating a new method(`capitalizeFirst`) and would also make the code simpler while still achieving the desired effect.
What do you think?

Reply:
@IAMOTZ I think I should try this format and see how all goes.